### ISSUE TITLE
Changing PWD breaks other parts of build system

### ISSUE THREAD

**Issue by konistehrad** [SamVerschueren/gulp-cordova-plugin#20]
At line 21 of `index.js` the process's PWD is updated. Updating the process PWD can blow up expectations of relative paths in other aspects of the build system. Is there a way to update this plugin such that it's not required? Thanks again for your hard work on this project.

**Comment by SamVerschueren** [comment]
Good point, have to check. If you have any idea on how to resolve this, PR is more then welcome :).

**Comment by SamVerschueren** [comment]
This is probably the cause of #1 as well.

**Comment by SamVerschueren** [comment]
@konistehrad Are you able to test if `process.chdir(file.path)` at line 21 solves the problem?
